.github/workflows/ci.yml

```
@@ -183,6 +183,49 @@ jobs:
           else
             echo "No changes detected"
           fi
+
+          # Revert git update-index --no-assume-unchanged for each entry
+          echo "Reverting assume unchanged for the following paths:"
+          for path in "${EXCLUDES[@]}"; do
+            echo "$path"
+            git update-index --no-assume-unchanged "$path"
+          done
+  sonar-cloud-quality-gate-status:
+    runs-on: ubuntu-latest
```

```suggestion
    if: github.event_name == 'pull_request'
    runs-on: ubuntu-latest
```

Adds the skip needed for merge-queue tested [here](https://github.com/MetaMask/metamask-mobile/actions/runs/10604420423)